docs/research/1995-codebase.md

```
@@ -45,22 +49,27 @@ All env vars set inside the image as defaults; can be overridden at container ru
 | `RACKULA_TRUST_PROXY` | `0` (unset) | Honor X-Forwarded-Proto from reverse proxy | `0`, `1`, `true`, `yes` (case-insensitive) |
 
 **nginx envsubst filter (line 53):** Only these vars are substituted into nginx template:
+
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)